src/App.js

```diff
@@ -9,7 +11,11 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article
+    title={article.title}
+    text={article.text}
+    date={article.date}
+  />
```

Comment:
використай спред оператор щоб передати дані в Article
```suggestion
  <Article {... article} />
```

Reply:
Денис, можно мне оставить в этом задании такой код?
В последующих заданиях уже использую деструктуризацию. Просто написал для большего понимания.